packages/framework/store/src/store/addon/indexer.ts

```diff
@@ -26,13 +26,24 @@ export const indexer = addOnFactory<keyof IndexerAddon>(
         super(storeOptions);
         const blockIndexer = new BlockIndexer(this.doc, { slots: this.slots });
         this.indexer = {
-          search: new SearchIndexer(this.doc),
+          search: !storeOptions.disableSearchIndex
+            ? new SearchIndexer(this.doc)
+            : null,
           backlink: new BacklinkIndexer(blockIndexer),
```

Comment:
Will the new indexer also replace the backlink here? I think maybe the flag can also control the backlink indexer.

Reply:
I will support backlink later